static_inventories/logs_library.yml

```diff
@@ -64,8 +64,37 @@ all:
               security_group: "{{ stack_prefix }}_logservers"
               assign_floating_ip: true
               add_hostname_to_ip_address: true
+    repo:
+      hosts:
+        repo-primary:
+          cluster_class: 'development'
+          azure_flavor: "Standard_DS1_v2"
+          host_networks:
+            - name: "{{ stack_prefix }}_external"
+              security_group: "{{ stack_prefix }}_repository"
+              assign_floating_ip: true
+              add_hostname_to_ip_address: true
+          local_volume_extra:
+            - name: mnt
+              size: 500
+        repo-secondary:
+          cluster_class: 'development'
+          cloud_flavor: umcg.v1.vm.1-2
+          host_networks:
+            - name: "{{ stack_prefix }}_internal_management"
+              security_group: "{{ stack_prefix }}_repository"
+              assign_floating_ip: true
+              add_hostname_to_ip_address: true
+          local_volume_extra:
+            - name: mnt
+              size: 500
+cluster:
```

Comment:
This would apply all roles defined for `cluster` machines to `logs` servers.... Some should be applied to logs servers too, others not.